rotkehlchen/tests/api/test_liquity.py

```diff
@@ -226,3 +226,47 @@ def mocked_request_dict(url, *_args, **_kwargs):
     assert result[JUSTIN]['deposited']['asset'] == A_LUSD
     assert FVal(result[JUSTIN]['deposited']['amount']) == expected_amount
     assert FVal(result[JUSTIN]['deposited']['usd_value']) == expected_amount * FVal(1.5)
+
+
+@pytest.mark.parametrize('use_custom_database', ['liquity_stats.db'])
+@pytest.mark.parametrize('perform_upgrades_at_unlock', [False])
+@pytest.mark.parametrize('new_db_unlock_actions', [None])
+@pytest.mark.parametrize('ethereum_modules', [['liquity']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+@pytest.mark.parametrize('add_account_to_db', [[False]])
+@pytest.mark.parametrize('ethereum_accounts', [
+    [
+        '0xF662f831361c8Ab48d807f7753eb3d641be25d24',
+        '0xbB8311c7bAD518f0D8f907Cad26c5CcC85a06dC4',
+    ],
+])
+def test_staking_stats(rotkehlchen_api_server, ethereum_accounts):
```

Comment:
Shouldn't this test be vcr'ed ?

Reply:
No external query is made. If you look at it is a prepared database that already contains the events and what it does is verify that the results are what I've verified by manually checking them. There are multiple events and adding them manually would be slow

Reply:
Yes, makes sense :+1: